coalib/output/ConsoleInteraction.py

```diff
@@ -560,26 +562,35 @@ def choose_action(console_printer, actions):
     :return:                Return choice of action of user.
     """
     while True:
-        console_printer.print(format_lines('*0: ' +
-                                           'Do nothing'))
+        dict_actions = {'(O)pen file': 'O', '(A)pply patch': 'A',
+                        'Print (M)ore info': 'M', 'Add (I)gnore comment':
+                            'I', 'Print (D)ebug message': 'D',
+                            'Print Aspec(T) Information': 'T'}
```

Comment:
can you be more specific on "automatically determine the right character from the documentation"? I don't understand